site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "minikube start"
```

Comment:
This shoudl be `setup minikube in China`

Reply:
sure

Reply:
updated.